src/Walker/Collector.C

```diff
@@ -29,69 +29,13 @@ CkReduction::reducerType PDFMerger;
 using walker::Collector;
 
 void
-Collector::chareOrd( const std::vector< tk::real >& ord )
+Collector::chareOrd( const std::vector< tk::real >& ord,
```

Comment:
I agree, both are pretty heinous.

Have you thought about how you would _talk_ **in person** about this code? (Would that be harder and more time-wasting with the longer function names?)